frontend/src/app/main-table/resource-table/resource-table.component.ts

```diff
@@ -26,14 +41,55 @@ export class ResourceTableComponent implements OnInit {
   ];
   dataSource = new MatTableDataSource();
 
+  showNameFilter = false;
+
   constructor(
+    private namespaceService: NamespaceService,
+    private k8s: KubernetesService,
     private dialog: MatDialog
-  ) {}
+  ) { }
+
+  ngOnInit() {
+    this.dataSource.sort = this.sort;
+
+    // Create the exponential backoff poller
+    this.poller = new ExponentialBackoff({ interval: 2000, retries: 3 });
```

Comment:
This component isn't supposed to fetch data. Should adding the cost table be modifying the `resource-table` at all?